packages/salesforcedx-vscode-lwc/package.nls.json

```diff
@@ -1,7 +1,7 @@
 {
-  "force_lightning_lwc_start_text": "SFDX: Start Component Preview",
-  "force_lightning_lwc_stop_text": "SFDX: Stop Component Preview",
-  "force_lightning_lwc_open_text": "SFDX: Open Component Preview",
+  "force_lightning_lwc_start_text": "SFDX: Start LWC Code Preview server",
+  "force_lightning_lwc_stop_text": "SFDX: Stop LWC Code Preview server",
+  "force_lightning_lwc_open_text": "SFDX: Open Code Preview Home",
```

Comment:
@LisandroFernandezSF - looking at all the palette commands, all words are initial-capitalized. At least I didn't see any not. So that's why I capitalized "server"

cc: @lcampos - yes? @smaddox-sf

Reply:
Update: cap'd except for prepositions.